README.md

```diff
@@ -4,7 +4,12 @@ This project allows users to create word cloud questions. Answers to each questi
 inverted Wordle that highlights minority answers.
 
 The front end of the project is built with [Eleventy](https://11ty.dev/). The website is deployed on
-[Netlify](https://www.netlify.com/).
+[Netlify](https://www.netlify.com/). The wordle data is saved in [the inclusive-design/inverted-wordles GitHub
+repository](https://github.com/inclusive-design/inverted-wordles/).
+
+When a new wordle question is created or an existing wordle question is modified, the new/updated question is pushed
+into the GitHub repository that triggers Netlify to deploy/re-deploy the wordle website. The detail of the architecture
+of this project can be found at [Inverted Wordles Architecture wiki page](https://wiki.fluidproject.org/display/fluid/Inverted+Wordles+Architecture).
```

Comment:
The README should have instructions giving at least an outline of end to end instructions on how to enlist one's own clone of the repo into netlify so that they can be tested in a self-contained way.
The steps that I found include - 
- Enrolling in netlify
- Choosing a plan
- Creating a team
- Enlisting a site pointing at a repo
- Correctly setting the deploy settings - e.g. "deploy all branches"
- Allocating one's own tokens at both github and netlify
- Setting environment variables in netlify's settings for the site
- Enlisting the site in Netlify identity
- (probably) picking at least one external provider in Netlify identity, e.g. GitHub to use one's own account
- .... anything else?